data/snippet/bulk_origin_proxies.toml

```
@@ -0,0 +1,30 @@
+name = "Bulk Origin Proxies"
+description = "Proxy third-party origins on your domain in bulk"
+code = """
+async function handleRequest(request) {
+  let url = new URL(request.url)
+  // Check if incoming hostname is
+  // a key in the origins object
+  let target = origins[url.hostname]
+  // If it is, proxy data from 
+  // third party origin and send it
+  if (target) {
+    url.hostname = target
+    return fetch(url, request)
+  }
+  // Otherwise, process request as normal
+  return fetch(request)
+}
+addEventListener('fetch', event => {
+  event.respondWith(handleRequest(event.request))
+})
+/**
+ * An object with different url's to fetch
+ * @param {Object} origins
+*/
+const origins = {
```

```suggestion
const ORIGINS = {
```
Starting to now all caps constants passed in. Also put them at the top please. (I know I said opposite before. Changed it b/c this follows normal JS convention and as long as small variable this doesn't obstruct the important code"